hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/prehandle/PreHandleWorkflowImpl.java

```diff
@@ -234,6 +222,21 @@ private PreHandleResult expandAndVerifySignatures(
                     "Payer account disappeared between preHandle and preHandleContext creation!", preCheck);
         }
 
+        // 2. Expand the Payer signature
+        final Key payerKey;
+        if (!isHollow(payerAccount)) {
+            // If the account IS a hollow account, then we will discover all such possible signatures when expanding
+            // all "full prefix" keys above, so we already have it covered. We only need to do this if the payer is
+            // NOT a hollow account (which is the common case).
+            payerKey = payerAccount.keyOrThrow();
+            signatureExpander.expand(payerKey, originals, expanded);
+        } else {
+            payerKey = null;
```

Comment:
Since `0.0.800` payer will now make it into this block, does it make sense to add,
```
if (EMPTY_KEY_LIST.equals(payerAccount.keyOrThrow())) {
    return preHandleFailure(payer, EMPTY_KEY_LIST, INVALID_PAYER_ACCOUNT_ID, txInfo, Set.of(), Set.of(), Map.of());
}
```
? Or I guess we're safe because the `EMPTY_KEY_LIST` will never have an OK `Verifiaction`?

Reply:
If someone mentioned 800 staking reward account as payer